src/server_manager/install_scripts/install_server.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Again, let's revert this - it probably is safe but I'd leave it for a separate PR.